### PR TITLE
ZIP-45: filenames not being sorted properly into multiple sequence sets

### PR DESCRIPTION
**Pull request by lbjay**
tbh I don't actually know what was wrong with the existing code. it was working very hard tho and resulting in files not getting sorted properly into correct sequence sets.

### CODE REVIEW

File: functions/zoom-downloader.py [hunk 1]
```diff
@@ -494,7 +495,7 @@ def stream(self):
 
     @property
     def file_extension(self):
-        return self.zoom_filename.split(".")[-1]
+        return splitext(self.zoom_filename)[-1]
```
Comment (nataliemona) [inline functions/zoom-downloader.py]: This is creating a filename with two "."
Like so:
"577132153/1585344688/001-shared_screen_with_speaker_view..mp4"

File: functions/zoom-downloader.py [hunk 2]
```diff
@@ -131,33 +132,33 @@ def host_name(self):
...
-                        ZoomFile(file_data, track_sequence)
-                    )
+                track_seq = start_times.index(file["recording_start"])
+                file["meeting_uuid"] = self.uuid
+                file["zoom_series_id"] = self.zoom_series_id
+                file["created_local"] = self._created_local
+                zoom_files.append(ZoomFile(file, track_seq))
+
+            self._recording_files = zoom_files
+
```
Comment (nataliemona) [inline functions/zoom-downloader.py:161]: Would be nice to have:
logger.info(self._recording_files)
here

File: functions/zoom-downloader.py [hunk 3]
```diff
@@ -131,33 +132,33 @@ def host_name(self):
...
             files = self.data["recording_files"]
-            start_times = sorted(
-                set([file["recording_start"] for file in files])
-            )
-            track_numbers = {
-                start_times[i]: i for i in range(len(start_times))
-            }
+            start_times = sorted(set(
+                [ file["recording_start"] for file in files ]
```
Comment (nataliemona) [inline functions/zoom-downloader.py]: The spaces around the brackets are a linter error for me

**Review by nataliemona — CHANGES_REQUESTED** [review]

File: functions/zoom-downloader.py (hunk 1 above)
Comment (nataliemona) [reply functions/zoom-downloader.py]: You can just remove the period in the s3_filename function

Comment (lbjay) [reply functions/zoom-downloader.py]: oh wow good catch. i did not think it worked that way. i shouldn't be making unrelated changes like this anyway.

File: functions/zoom-downloader.py (hunk 2 above)
Comment (lbjay) [reply functions/zoom-downloader.py:161]: it's just going to print a bunch of `<ZoomFile...>` items, no?. You could make a separate PR to define a `__repr__` method for the ZoomFile class, and then add the log event.

File: functions/zoom-downloader.py (hunk 3 above)
Comment (lbjay) [reply functions/zoom-downloader.py]: how have I done this for so long without the pylint plugin installed?

**Pull request merged**